stargazer/stargazer.py

```diff
@@ -176,6 +178,14 @@ def reset_covariate_order(self):
         if self.original_cov_names is not None:
             self.cov_names = self.original_cov_names
 
+    def add_custom_lines(self, lines, location='footer top'):
```

Comment:
+1 to @toobaz's comment, I think `add_line` would be more natural.

Also `lines` seems like an odd term, isn't this just one line that's added at a time? Below when you loop through the elements, those are individual values right?